django_scrubber/__init__.py

```diff
@@ -11,6 +12,17 @@
     'SCRUBBER_ADDITIONAL_FAKER_PROVIDERS': [],
     'SCRUBBER_FAKER_LOCALE': None,
     'SCRUBBER_MAPPING': dict(),
+    'SCRUBBER_STRICT_MODE': False,
+    'SCRUBBER_REQUIRED_FIELD_TYPES': (models.CharField, models.TextField, models.URLField, models.JSONField,
+                                      models.GenericIPAddressField, models.EmailField),
```

Comment:
Can you please add a trailing comma? This should reformat and move all items in their own line.